JobConfig/common/trigger_templates.fcl

```diff
@@ -0,0 +1,260 @@
+# -*- mode: tcl -*-
```

Comment:
This file replicates a lot of existing functionality, it probably conflicts with other files in primary, reco, etc and should not be in 'common'.  I suggest moving to JobConfig/su2020.

Reply:
I've removed this file.